packages/block-library/src/latest-posts/index.php

```diff
@@ -151,6 +143,24 @@ function render_block_core_latest_posts( $attributes ) {
 
 			$trimmed_excerpt = get_the_excerpt( $post );
 
+			/*
+			 * Adds a "Read more" link with screen reader text.
+			 * [&hellip;] is the default excerpt ending from wp_trim_excerpt() in Core.
+			 */
+			if ( str_ends_with( $trimmed_excerpt, ' [&hellip;]' ) && 'excerpt' === $attributes['displayPostContentRadio'] ) {
```

Comment:
I haven't tested this PR yet, so this is just a drive-by comment! 😄

The condition above appears to already check for `&& 'excerpt' === $attributes['displayPostContentRadio'] )` so I don't think we need that condition in this `if` statement?

Reply:
Did I leave that in there? 😅 

Thanks for spotting 🙇🏻

Reply:
One more comment here: I'm not 100% convinced of the robustness of this check. We're hard-coding what we expect the default value to be (from Core). 

If a theme has added a hook that returns ` [&hellip;]`, which we presume is what they want then this condition will also overwrite.